packages/styles/scss/patterns/blocks/content-block-simple/README.md

```diff
@@ -0,0 +1,38 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-simple/index';
+```
+
+#### Modifiers
+
+Use these modifiers with `.bx--content-block-simple` class.
+
+| Class                                    | Description                                            |
+| ---------------------------------------- | ------------------------------------------------------ |
+| `.bx--content-block-simple--icon-link`   | The class name for simple long form with icon link     |
+| `.bx--content-block-simple--card`        | The class name for simple long form with card link     |
+| `.bx--content-block-simple--with-border` | The class name for simple long form with border bottom |
+
+#### Elements
+
+| Class                                    | Description                                                                     |
+| ---------------------------------------- | ------------------------------------------------------------------------------- |
+| `.bx--content-block-simple__container`   | Element containing all simple long form content                                 |
+| `.bx--content-block-simple__row`         | Vertical divisions across the viewport                                          |
+| `.bx--content-block-simple__col`         | Horizonal divisions across the viewport and change in width at different screen |
+| `.bx--content-block-simple__title`       | Title for simple long form pattern                                              |
+| `.bx--content-block-simple__content`     | Content holder for short description                                            |
+| `.bx--content-block-simple__link`        | Wrapper for link component                                                      |
+| `.bx--content-block-simple__divider_col` | Horizonal division for divider element                                          |
+| `.bx--content-block-simple__divider`     | Divider element                                                                 |
```

Comment:
Looks like in general this file needs to be updated. I don't think most of these are used.